course/templates/course/_course_menu.html

```diff
@@ -14,11 +14,10 @@
 </li>
 
 {% if instance.language|first == "|" %}
-</li>
-<li role="presentation" class="header visible-xs">
+<li class="header visible-xs">
 	<h4 id="course-menu-heading">{% trans "Language" %}</h4>
```

Comment:
I wonder why the subtitle "Language" has the id `course-menu-heading`. This is probably a mistake. This id is already used in the heading above the menu. @jaguarfi could you fix that? The language heading should use a different id.